src/TimeoutTrait.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace React\SocketClient;
+
+use React\EventLoop\LoopInterface;
+use React\Promise\CancellablePromiseInterface;
+
+trait TimeoutTrait
+{
+    protected function setTimeout(LoopInterface $loop, CancellablePromiseInterface $promise, $seconds = 30)
```

Comment:
Is much gained from having a default value for `$seconds` here? Any value is pretty arbitrary.